### DISCUSSION TITLE
Flows: send mail and send notification recipient

### DISCUSSION THREAD

**Discussion by mslonina** [directus/directus#13915] [Archive]
Hi,

is it possible to use variables in send mail/notification recipients? It could be taken for example from another flow step, like content item owner - to notify them, i.e. when the content has moved from a 'review' state to 'published'.

**Comment by rijkvanzanten** [comment]
Yeah! Every option in every operation accepts the `{{}}` syntax to pull in data from other operations, including the send mail `recipients` 🙂
